modules/015-admission-policy-engine/crds/doc-ru-operation-policy.yaml

```
@@ -31,6 +31,13 @@ spec:
                           description: "Список запросов на выделение ресурсов, которые должны быть выполнены (процессор, память или оба запроса)."
                     disallowedImageTags:
                       description: "Требует, чтобы образы контейнера имели тег, отличный от указанных в списке."
+                    disallowedTolerations:
+                      description: |
+                        Запрещает tolerations из указанного списка.
```

```suggestion
                        Запрещает создание подов с tolerations из указанного списка.
```
According to the English description